docs/hoon/library/1b.md

```diff
@@ -86,25 +139,88 @@ Examples
 
     > (mas 3)
     1
+
     > (mas 4)
     2
+
     > (mas 5)
     3
-    > (mas 6)
-    2
-    > (mas 0)
-    ! exit
+    > (cap 5)                                    ::`(cap a)` computes whether address `a` is in the head or the tail
+    %2
+
+    > (mas 7)
+    3
+    > (cap 7)
+    %3
+
+    > (mas 11)
+    7
+
+    > (mas (mas 11))
+    3
+
+    > (mas 'p')
+    47
+    > `@`'p'                                    
+    111
+
+    > (cap (mas 5))
+
+    > (mas 0)                                              
+    ! exit                                       ::address `0` is in neither the head nor the tail
+
     > (mas 1)
```

Comment:
I'm not sure what some of these examples are trying to show.  What's the use of `mas`ing a cord?